templates/default/consul-systemd.erb

```diff
@@ -6,12 +6,13 @@ After=basic.target network.target
 [Service]
 User=<%= node['consul']['service_user'] %>
 Group=<%= node['consul']['service_group'] %>
+EnvironmentFile=-<%= node['consul']['etc_config_dir'] %>
 ExecStart=<%= Chef::Consul.active_binary(node) %> agent \
   -config-dir <%= node['consul']['config_dir'] %>
 ExecReload=/bin/kill -HUP $MAINPID
 KillMode=process
 Restart=on-failure
-RestartSec=42s
+RestartSec=15s
```

Comment:
42 seemed like an arbitrary number (bring a towel, meaning of life, etc...).  15s is more in line with what I'd expect for a recovery time.